src/Symfony/Bundle/WebProfilerBundle/Tests/Resources/IconTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Tests\Resources;
+
+use PHPUnit\Framework\TestCase;
+
+class IconTest extends TestCase
+{
+    /**
+     * @dataProvider provideIconFilePaths
+     */
+    public function testIconFileContents($iconFilePath)
+    {
+        $this->assertRegExp('/<svg version="1.1" xmlns="http:\/\/www.w3.org\/2000\/svg" x="0px" y="0px" width="\d+" height="\d+" viewBox="0 0 \d+ \d+" enable-background="new 0 0 \d+ \d+" xml:space="preserve">.*<\/svg>/s', file_get_contents($iconFilePath), sprintf('The SVG metadata of the %s icon is different than expected (use the same as the other icons).', $iconFilePath));
```

Comment:
@javiereguiluz: okay now i understand why you use regexp.
hm but if we/you guys want to do that you could check if the width and hight does correspond to the viewBox

i was just thinking that we only need to check the head of the svg
hm should we also test if the svg is valid xml?

PS: we could also add xml declaration like this
```xml
<?xml version="1.0" encoding="utf-8"?>
```
or can't we do that because its embedded?